gravitee-apim-plugin/gravitee-apim-plugin-entrypoint/gravitee-apim-plugin-entrypoint-webhook/src/main/java/io/gravitee/plugin/entrypoint/webhook/exception/UnauthorizedException.java

```diff
@@ -13,18 +13,18 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package io.gravitee.plugin.entrypoint.webhook.configuration;
+package io.gravitee.plugin.entrypoint.webhook.exception;
 
-import lombok.Getter;
-import lombok.Setter;
+import io.gravitee.gateway.jupiter.api.exception.MessageProcessingException;
 
 /**
- * @author David BRASSELY (david.brassely at graviteesource.com)
  * @author GraviteeSource Team
  */
-@Getter
-@Setter
-public class Retry {
+public class UnauthorizedException extends MessageProcessingException {
 
-    private int attempts = 1;
+    private static final long serialVersionUID = -931625532200074753L;
+
+    public UnauthorizedException() {
+        super("Webhook entrypoint response status is 401 unauthorized");
```

Comment:
Authentication error could also lead to 403 I guess. I would be more general in the message error.